frontend/app/root.tsx

```diff
@@ -38,6 +39,8 @@ export async function loader({ request }: LoaderFunctionArgs) {
         buildVersion: '0.0.0-00000000-0000',
       },
       env: publicEnv,
+      sessionTimeoutPrompt: privateEnv.SESSION_TIMEOUT_PROMPT_SECONDS,
+      sessionTimeout: privateEnv.SESSION_TIMEOUT_SECONDS,
```

Comment:
These two values are going to be exposed publicly when the `<SessionTimeout />` renders. I feel there's no need to create a new `privateEnv` within the `loader()` to pass it to the `<SessionTimeout/>`.

We could alternatively take advantage of `env-utils`' `getClientEnv()` instead. For example:
https://github.com/DTS-STN/canadian-dental-care-plan/blob/aabe531e546f6adbccbb515a34b3e2037edc0327/frontend/app/components/language-switcher.tsx#L22

Reply:
The `getClientEnv()` function is most useful when you need to pass config to a non-route component, but don't want to pass the value down through a bunch of nested components to get there.

In cases where we're rendering a component directly in a route, I think using `.getEnv()` inside a loader to pass that config to the route, and then the target component is okay.

One advantage to using a `.getClientEnv()` call inside a route would be to avoid blocking rendering while the loader data is being fetched.. but since we have to do an auth check for every route, this becomes moot.